contribute/developer-guide.md

```diff
@@ -33,61 +33,61 @@ We recommend using the Git command-line interface to download the source code fo
+We use pre-commit hooks (via [lefthook](https://github.com/evilmartians/lefthook)) to lint, fix, and format code as you commit your changes. Previously, the Grafana repository automatically installed these hook when you ran `yarn install`, but they are now opt-in for all contributors.
 
-Install the lefthook precommit hooks with:
+To install the precommit hooks:
 
 ```sh
 make lefthook-install
 ```
 
-To remove precommit hooks, run
+To remove precommit hooks:
 
 ```sh
 make lefthook-uninstall
 ```
 
-> [!NOTE]
-> Contributors working on the frontend are highly encouraged to install the precommit hooks, even if your IDE formats on save, so the `.betterer.results` file is kept up to sync.
+> We strongly encourage contributors who work on the frontend to install the precommit hooks, even if your IDE formats on save. By doing so, the `.betterer.results` file is kept up to sync.
 
 ## Build Grafana
 
-Grafana consists of two components; the _frontend_, and the _backend_.
+When building Grafana, be aware that it consists of two components: the _frontend_, and the _backend_.
 
 ### Frontend
 
-Before we can build the frontend assets, we need to install the dependencies:
+Before you can build the frontend assets, you need to install the related dependencies:
 
 ```
 yarn install --immutable
 ```
 
-> Troubleshooting: if you get the error `The remote archive doesn't match the expected checksum` for a dependency pulled from a link (e.g. `"tether-drop": "https://github.com/torkelo/drop"`): this is a temporary mismatch. To work around it (while someone corrects the issue), you can prefix your `yarn install --immutable` command with [`YARN_CHECKSUM_BEHAVIOR=update`](https://yarnpkg.com/advanced/error-codes#yn0018---cache_checksum_mismatch)
+> If you get the error `The remote archive doesn't match the expected checksum` for a dependency pulled from a link (for example, `"tether-drop": "https://github.com/torkelo/drop"`): this is a temporary mismatch. To work around the error (while someone corrects the issue), you can prefix your `yarn install --immutable` command with [`YARN_CHECKSUM_BEHAVIOR=update`](https://yarnpkg.com/advanced/error-codes#yn0018---cache_checksum_mismatch).
 
-After the command has finished, we can start building our source code:
+After the command has finished, you can start building the source code:
 
 ```
 yarn start
 ```
 
-This command will generate sass theme files, build all external plugins, then build the frontend assets.
-Once `yarn start` has built the assets, it will continue to do so whenever any of the files change. This means you don't have to manually build the assets every time you change the code.
+This command generates SASS theme files, builds all external plugins, and then builds the frontend assets.
```

Comment:
SASS theme or SASS themed? (I don't actually know, just asking because by brain wants to write it that way, but I don't know if "theme files" are a thing?)

Reply:
I think original is correct.